Code-4-Community/speak-for-the-trees-frontend
# Create settings page

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/pulses/816231284)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

Settings page for users to edit their account settings or delete/deactivate their account, currently just styling

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

Updated Settings.tsx and created settings.less to match the designs
- Used styled components for form titles and submit buttons to override internal styles
- Used Col grids to space out the form input fields

## Screenshots

Provide screenshots of any new components, styling changes, or pages.
<img width="1680" alt="Settings" src="https://user-images.githubusercontent.com/22990100/100139571-ad581800-2e5d-11eb-9d21-90818782ae86.png">

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.
- Visually checked that the page matches the designs
- Shrunk/maximized window size to check flexibility
